active_model_serializers.gemspec

```diff
@@ -46,6 +46,8 @@ Gem::Specification.new do |spec|
     # activesupport
     # activemodel
 
+  spec.add_runtime_dependency 'json_schema'
```

Comment:
It's kind of interesting that this is a runtime dependency intended for usage in client tests.